src/main/resources/application.properties

```diff
@@ -45,4 +45,10 @@ quarkus.micrometer.export.prometheus.path=/metrics
 
 # Database Configuration
 azure.storage.connection-string=${AZURE_STORAGE_CONNECTION_STRING:UseDevelopmentStorage=true}
-azure.storage.table-name=${AZURE_STORAGE_TABLE_NAME:products}
+azure.storage.table-name=${AZURE_STORAGE_TABLE_NAME:tools}
+
+# PostgreSQL Configuration
+postgresql.url=jdbc:postgresql://localhost:5432/mydatabase
+postgresql.username=username
+postgresql.password=secret
```

Comment:
Gaat dit later naar env? If so, update dan de platform-config repo accordingly (maak PR in platform-config repo en message mij).

Reply:
Is momenteel for lokale access, om ff te kijken of alles werkt.

Reply:
Yes yes I understand, maar waarom is dat een argument? Als je je dingen alvast in een env zet heb je nog steeds local access, alleen zorg je er wel voor dat je dat alvast hebt ingesteld in de code (zorg er ook voor dat je het in de .env.example aangeeft).

Reply:
Oh en wil jij een Azure PostgreSQL instance op Azure aanmaken die we dan allemaal aanroepen of moet iemand anders dat doen?

Reply:
> Yes yes I understand, maar waarom is dat een argument? Als je je dingen alvast in een env zet heb je nog steeds local access, alleen zorg je er wel voor dat je dat alvast hebt ingesteld in de code (zorg er ook voor dat je het in de .env.example aangeeft).

Nee daar heb je een punt. Dan zet ik het wel ff in de env.example

Reply:
> Oh en wil jij een Azure PostgreSQL instance op Azure aanmaken die we dan allemaal aanroepen of moet iemand anders dat doen?

Kunnen we dat niet beter met meerdere doen? Anders ga ik binnenkort wel ff in een call met Bastiaan want als ik het goed begrijp werkt hij ook met PostgreSQL.

Reply:
ik heb het ook nodig

Reply:
Yes deze env kan in meerdere pods gebruikt worden. Check de platform-config repo en/of vraag mij even om hulp. Misschien goed idee om alvast een PostgreSQL instance op Azure aan te maken Daan?

Reply:
Ja ga ik morgen mee beginnen